### PR TITLE
Init/frontend

### PR DESCRIPTION
**Pull request by M4R1KU**
the frontend project is initialized and ready
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/outcobra/outstanding-cobra/pull/1%23discussion_r84041144%22%2C%20%22https%3A//github.com/outcobra/outstanding-cobra/pull/1%23issuecomment-254776245%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/outcobra/outstanding-cobra/pull/1%23issuecomment-254776245%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22LGTM%22%2C%20%22created_at%22%3A%20%222016-10-19T10%3A36%3A51Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/613138%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jmesserli%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20bc8ffa8f03b5c45f400aa8e0ce3c23f386b2a714%20frontend/angular-cli.json%2011%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/outcobra/outstanding-cobra/pull/1%23discussion_r84041144%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22should%20we%20add%20moment%20already%3F%22%2C%20%22created_at%22%3A%20%222016-10-19T10%3A33%3A50Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/613138%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jmesserli%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20frontend/angular-cli.json%3AL15-29%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull bc8ffa8f03b5c45f400aa8e0ce3c23f386b2a714 frontend/angular-cli.json 11'></a> &nbsp;<b><a href='https://github.com/outcobra/outstanding-cobra/pull/1#discussion_r84041144'>File: frontend/angular-cli.json:L15-29</a></b>
- <a href='https://github.com/jmesserli'></a> should we add moment already?
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/outcobra/outstanding-cobra/pull/1#issuecomment-254776245'>General Comment</a></b>
- <a href='https://github.com/jmesserli'></a> LGTM

<a href='https://www.codereviewhub.com/outcobra/outstanding-cobra/pull/1?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/outcobra/outstanding-cobra/pull/1?approve=1'></a>&nbsp;<a href='https://github.com/outcobra/outstanding-cobra/pull/1'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by jmesserli** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/outcobra/outstanding-cobra/pull/1'>Refresh</a>.<a href='#crh-update-none'></a>

File: frontend/angular-cli.json [hunk 1]
```diff
@@ -15,9 +15,14 @@
...
       "mobile": false,
       "styles": [
-        "styles.css"
+          "outcobra.scss",
+          "../node_modules/materialize-css/bin/materialize.css",
+          "../node_modules/material-design-icons/iconfont/material-icons.css"
+      ],
+      "scripts": [
+          "../node_modules/jquery/dist/jquery.js",
+          "../node_modules/materialize-css/bin/materialize.js"
```
Comment (jmesserli) [inline frontend/angular-cli.json]: should we add moment already?

**Comment by jmesserli** [comment]
LGTM

**Comment by jmesserli** [comment]
:shipit:

File: frontend/angular-cli.json (hunk 1 above)
Comment (M4R1KU) [inline frontend/angular-cli.json]: @jmesserli You don't have to add moment there. We just have to add a new dependency in the package.json and ensure that we have the TypeScript types installed and then the angular-cli will automatically bundle moment with all the other stuff.

Comment (jmesserli) [inline frontend/angular-cli.json]: good to know. all good then.

**Pull request merged**